data_architect_misc/data_transformer/configs/APAC_India/config.json

```diff
@@ -88,49 +62,40 @@
       {
         "__function_comment__": "Create HARMONIZED_MEDIA_TYPE column (with standardized Media Type names in comp_harm_constants.py) from raw media type column.",
         "function_name": "add_HARMONIZED_MEDIA_TYPE_column_using_existing_media_type_column",
-        "function_args": ["MEDIA_TYPE"]
+        "function_args": ["MEDIUM"]
       },
       {
         "__function_comment__": "Add CURRENCY as a new column with 'ZAR' as value.",
         "function_name": "add_HARMONIZED_CURRENCY_column",
-        "function_args": ["ZAR"]
+        "function_args": ["INR"]
       },
       {
         "__function_comment__": "Create HARMONIZED_GROSS_SPEND column (with values rounded to 2-decimal places) from raw gross spend column.",
         "function_name": "add_HARMONIZED_GROSS_SPEND_column",
-        "function_args": ["GROSS_SPEND_IN_LOCAL_CURRENCY"]
+        "function_args": [["TAM COST"]]
```

Comment:
This should be "Actual TAM Cost". If you decide to use 'TAM COST", then shouldn't you be multiplying this with 1000?